web-portal/internal-api/utils.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * Utilities for the internal API endpoints -- don't use these in SSR code
+ */
+
+// logging support (copied from api-server)
+const winston = require('winston')
+const { createLogger, format, transports } = winston
+const { combine, timestamp, label, printf, colorize } = format
+const env = process.env.ENV || 'dev'
```

Comment:
Does this need to be added to the `.env` file for the web portal?

Reply:
yeah that should be in there so you can log the environment. It's already there on the web portal servers

Reply:
if you don't have it set locally it will default to dev, which is what you would want anyyways